images/ceph/Makefile

```diff
@@ -60,7 +60,8 @@ do.build:
 	@cp -r $(MANIFESTS_DIR)/monitoring $(BUILD_CONTEXT_DIR)/ceph-monitoring
 	@mkdir -p $(BUILD_CONTEXT_DIR)/rook-external/test-data
 	@cp $(MANIFESTS_DIR)/create-external-cluster-resources.* $(BUILD_CONTEXT_DIR)/rook-external/
-	@cp ../../tests/ceph-status-out $(BUILD_CONTEXT_DIR)/rook-external/test-data/
+	@cp ../../tests/external-cluster/ceph-status-out $(BUILD_CONTEXT_DIR)/rook-external/test-data/
+	@cp ../../tests/external-cluster/external-config.ini $(BUILD_CONTEXT_DIR)/rook-external/
```

Comment:
This is adding the ini to the container image? Am I understanding right?

Reply:
That's correct, the files were added so I also added this one

Reply:
I think we need this in future when we need UI to show the config .ini

Reply:
Let's talk more about this. I don't think this should be necessary. 

Let's also talk about why `ceph-status-out` is being added to the container. I don't see any reason why it should be necessary to add test data to the container image. Test data should only be necessary in the dev environment.

The config file has 2 values filled in for rbd data pool name and rados namespace that likely aren't present in user clusters.

The `external-config.ini` file isn't being auto-generated anywhere. There is risk of it falling out of date over time, which I suspect will be in less than 9 months. If we are going to have an example (which I don't think we need), let's ensure that it won't become stale.

Reply:
Removed it